### PR TITLE
Advice

### PR DESCRIPTION
**Pull request by Karina-R**

### CODE REVIEW

**Comment by Karina-R** [comment]
Sorry for the same commits. I wasn't sure if I was merging correctly so I re-did the same steps over and over on the command line with the same commit. Everything should be correctly.

**Comment by acodeinprogress** [comment]
What great advice, @Karina-R!

This is looking excellent. The only thing is that you added a k next to <!DOCTYPE html>. Once you remove that, you can add, commit and push and then we can get this merged in.

xoxo,
Sharon

**Comment by Karina-R** [comment]
Hi! You didn't specify where that k was, but I think I found it and removed it. If not, please let me know! Thanks!

File: advice/karina-r-advice.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
```
Comment (Karina-R) [inline advice/karina-r-advice.html]: This is where I found the k before <!DOCTYPE html> and removed it.

**Comment by redrambles** [comment]
Hey @Karina-R,

Thanks for making that change! Your code is all merged in.

Cheers,
Ann :)

**Pull request merged**